getsentry/sentry
# ref(performance): Add sampling alert Call to action  MVP - [TET-430]

**MVP Preview:**


https://user-images.githubusercontent.com/29228205/193799308-5e5972f8-be63-4f2d-a2bc-7550c4f67822.mov



**What still has to be done in this PR:**

- Add the feature flag to this logic ✅
- Add the logic to see if the diff between the current and recommended samples is equal to or more than 50%  ✅
- Test, Test... Test  ✅
- The PR https://github.com/getsentry/sentry/pull/39508 has to be merged first as it contains the route changes  
- Check if we display the alert "Project incompatible with sampling" and if that will be the case, we won't display this alert   ✅
- Add analytics code ✅
